README.md

```diff
@@ -10,6 +10,21 @@
 Docker Swarm is native clustering for Docker. It turns a pool of Docker hosts
 into a single, virtual host.
 
+## Swarm Disambiguation
+
+**Docker Swarm standalone**: This project. An API proxy system in which a single Docker host fronts several Docker Engines to make them appear as if they are a single Docker Engine.
+See [Docker Swarm overview](https://docs.docker.com/swarm/overview/).
+
+**[Swarmkit](https://github.com/docker/swarmkit)**: Cluster management and
+orchestration features in Docker Engine 1.12 or later. When you enable this
+plugin, the Docker Engine runs swarm mode. Swarm mode includes the features
+listed in the [Swarm mode overview](https://docs.docker.com/engine/swarm/).
+
+**[Docker Engine swarm mode](https://github.com/docker/docker)**: CLI interface
```

Comment:
@nishanttotla , part of the problem here is that the current doc structure only supports 1 link here.  updated this to the swarm api because it seems more directly related